src/client/pages/_slug.vue

```diff
@@ -24,3 +36,43 @@ export default {
   computed: {},
 }
 </script>
+
+<style>
+@import '../components/app-core/index.css';
+
+.page__header {
+  max-width: 100%;
+  margin-bottom: 4rem;
+}
+
+.page__content {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  padding: 0 var(--spacing-default);
+  max-width: 920px;
+}
+
+.page__content__sidebar {
+  width: 300px;
+  margin-right: var(--spacing-double);
+}
+
+.page__content__main-content {
+  max-width: 600px;
+}
+
+@media screen and (max-width: 880px) {
```

Comment:
following the mobile first principle, you should start with styling for small screen sizes and use media queries for larger screen sizes, using `min-width`